test/unittests/Microsoft.ServiceFabric.Actors.Wcf.Tests/WcfActorServiceRemotingListenerTest.cs

```diff
@@ -0,0 +1,100 @@
+                var mockCodePackageActivationContext = Mock.Of<ICodePackageActivationContext>();
+                Mock.Get(mockCodePackageActivationContext).Setup(c => c.GetEndpoints()).Returns(mockEndpointsCollection);
+                Mock.Get(mockCodePackageActivationContext).Setup(c => c.ApplicationName).Returns("fabric:/DummyApp");
+                Mock.Get(mockCodePackageActivationContext).Setup(c => c.ApplicationTypeName).Returns("fabric:/DummyApp");
+
+                var statefulServiceContext = new StatefulServiceContext(
+                    new NodeContext("NodeName", new NodeId(0, 1), 0, "NodeType", "127.0.0.1"),
+                    mockCodePackageActivationContext,
+                    "DummyServiceType",
+                    new Uri("fabric:/DummyApp/DummyService"),
+                    null,
+                    Guid.NewGuid(),
+                    1L
+                );
+
+                ActorTypeInformation actorTypeInfo = ActorTypeInformation.Get(typeof(DummyActor));
+
+                ActorService dummyActorService = new ActorService(
+                    statefulServiceContext,
+                    actorTypeInfo,
+                    null,
+                    null,
+                    null,
+                    new ActorServiceSettings());
+
+                // Act
+                var listener = new WcfActorServiceRemotingListener(
+                    actorService: dummyActorService,
+                    listenerBinding: null,
+                    useWrappedMessage: false,
+                    exceptionConvertors: exceptionConvertors,
+                    settings: settings
+                );
+
+                // Assert
+                WcfRemotingService wcfRemotingService = listener.Field<WcfRemotingService>().Value;
+                ExceptionSerializer actualSerializer = wcfRemotingService.Field<ExceptionSerializer>().Value;
+
+                IEnumerable<IExceptionConvertor> actualConvertors = actualSerializer.Field<IEnumerable<IExceptionConvertor>>().Value;
+                Assert.NotNull(actualConvertors);
+
+                var convertorList = new List<IExceptionConvertor>(actualConvertors);
+                Assert.Equal(5, convertorList.Count);
+                Assert.IsType<SystemExceptionConvertor>(convertorList[0]); // passed in
+                Assert.IsType<FabricActorExceptionConvertor>(convertorList[1]); // default
+                Assert.IsType<SystemExceptionConvertor>(convertorList[2]); // default
+                Assert.IsType<FabricExceptionConvertor>(convertorList[3]); // default
+                Assert.IsType<DefaultExceptionConvertor>(convertorList[4]); // default
+
+                var actualSettings = actualSerializer.Field<FabricTransportRemotingListenerSettings>().Value;
+                Assert.Same(settings, actualSettings);
+            }
+        }
+    }
+
+    public interface IDummyActor : IActor { }
```

Comment:
I would recommend defining test helpers like IDummyActor as nested types inside the top-level class to avoid accidental reuse by other tests. This also keeps the code with the one file per type rule.